controller/wingsail/wingsail_ctrl_node.py

```diff
@@ -90,6 +109,14 @@ def pub_period(self) -> float:
     def trim_tab_angle(self) -> float:
         return self.__trim_tab_angle
 
+    def __filtered_wind_sensor_sub_callback(self, msg):
+        self.filtered_wind_sensor = msg
+        self.get_logger().info(f"Received data from {self.__filtered_wind_sensor_sub.topic}")
```

Comment:
Make sure to add types to your inputs and outputs. For example, this function should look something like this:

```python
def __filtered_wind_sensor_sub_callback(self, msg: WindSensor):
    ...
```

Notice how there is no return type specified. This is because this function returns nothing (i.e. `None`). It is technically okay to put this as well (but not necessary):

```python
def __filtered_wind_sensor_sub_callback(self, msg: WindSensor) -> None:
    ...
```

You can tell if you typed correctly if the `mypy` task passes (either locally for you, or on the CI that you see on your PR).

This feedback is similar for `__gps_sub_callback`.

Reply:
Also, be sure to add docstrings to these callbacks to document what they do. Do this after typing, because the VS Code extension that we installed should automatically document the types and variable names, you just need to fill in the descriptions.

Reply:
I typed and added input types to callback functions and added docstrings as well, please resolve if you agree with what I added.